Makefile

```diff
@@ -230,6 +230,7 @@ prepare-test-env:
 
 	echo "Step 1: Setting up the test environment..."; \
 	$(MAKE) up; \
+	./bin/config.sh int MaxObjectSize 1024; \
```

Comment:
since that is a helper and it _already_ uses `$(MAKE)` can it be `$(MAKE) update.max_object_size val=1000`?

Reply:
No. I've specifically tried to avoid calling `$(MAKE)` again. `$(MAKE) update.max_object_size val=1000` is an über-kludge, we've misusing `make` (which is _not_ a set of shell scripts) right and left, but I don't like adding more mess into it.